docs/contributing-to-airbyte/tutorials/building-a-python-destination.md

```diff
@@ -0,0 +1,193 @@
+# Building a Python Destination
+
+## Summary
+
+This article provides a checklist for how to create a Python destination. Each step in the checklist has a link to a more detailed explanation below.
+
+## Requirements
+
+Docker and Python with the versions listed in the [tech stack section](../../understanding-airbyte/tech-stack.md). You can use any Python version between 3.7 and 3.9, but this tutorial was tested with 3.7.  
+
+## Checklist
+
+### Creating a destination
+
+* Step 1: Create the destination using the template generator
+* Step 2: Setup the virtual environment 
+* Step 3: Implement `spec` to define the configuration required to run the connector
+* Step 4: Implement `check` to provide a way to validate configurations provided to the connector
+* Step 5: Implement `write` to write data to the destination
+* Step 6: Set up Acceptance Tests
+* Step 7: Write unit tests or integration tests
+* Step 8: Update the docs \(in `docs/integrations/destinations/<destination-name>.md`\)
+
+{% hint style="info" %}
+If you need help with any step of the process, feel free to submit a PR with your progress and any questions you have, or ask us on [slack](https://slack.airbyte.io). Also reference the KvDB python destination implementation if you want to see an example of a working destination. 
+{% endhint %}
+
+## Explaining Each Step
+
+### Step 1: Create the destination using the template
+
+Airbyte provides a code generator which bootstraps the scaffolding for our connector.
+
+```bash
+$ cd airbyte-integrations/connector-templates/generator # assumes you are starting from the root of the Airbyte project.
+$ ./generate.sh
+```
+
+Select the `Python Destination` template and then input the name of your connector. We'll refer to the destination as `destination-<name>` in this tutorial, but you should replace `<name>` with the actual name you used for your connector e.g: `redis` or `google-sheets`. 
+
+### Step 2: Setup the dev environment
+
+Setup your Python virtual environment: 
+
+```bash
+cd airbyte-integrations/connectors/destination-<name>
+# Create a virtual environment in the .venv directory
+python -m venv .venv 
+# activate the virtualenv
+source .venv/bin/activate 
```

Comment:
don't think this is necessary for this PR but could also include Windows translations of setup bash commands for Win devs at some point

Reply:
https://github.com/airbytehq/airbyte/issues/4832